test/index.js

```diff
@@ -555,8 +555,7 @@ test('strict: when short option and suspect value then throws with whole expecte
 
   assert.throws(() => {
     parseArgs({ args, options });
-  // eslint-disable-next-line max-len
-  }, /Error: Option '-w' argument is ambiguous\.\nDid you forget to specify the option argument for '-w'\?\nOr to specify an option argument starting with a dash use '--with=-XYZ' or '-w-XYZ'\./
+  }, /To specify an option argument starting with a dash use '--with=-XYZ' or '-w-XYZ'\./
```

Comment:
Minor, and can safely be ignored. I deliberately had a couple of tests of the "whole" error message. But just take `whole` out of the test name and I'll be happy. 😄

Reply:
The error message string varies a bit in Node.js.

If you don't mind my preference is the slightly shorter comparison.